CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.5)
 project(eosio_contracts)
 
 set(VERSION_MAJOR 3)
-set(VERSION_MINOR 3)
+set(VERSION_MINOR 2)
```

Comment:
did you mean to downgrade this number?

Reply:
This should be 3.3. My mistake the reference-contract PR with version bump was stuck, and I pulled the wrong version in.